frontend/lib/src/components/elements/ArrowVegaLiteChart/ArrowVegaLiteChart.test.tsx

```diff
@@ -60,30 +63,36 @@ const getProps = (props: Partial<PropsWithHeight> = {}): PropsWithHeight => ({
   ...props,
 })
 
+const getCanvasContext = (): CanvasRenderingContext2D => {
+  const canvas = screen
+    .getByTestId("stArrowVegaLiteChart")
+    // eslint-disable-next-line testing-library/no-node-access
+    .querySelector("canvas") as HTMLCanvasElement
+
+  return canvas.getContext("2d") as CanvasRenderingContext2D
+}
+
 describe("VegaLiteChart Element", () => {
-  it("renders without crashing", () => {
+  it("renders without crashing", async () => {
     const props = getProps()
-    const wrapper = mount(<ArrowVegaLiteChart {...props} />)
 
-    expect(wrapper.find("StyledVegaLiteChartContainer").length).toBe(1)
+    await render(<ArrowVegaLiteChart {...props} />)
```

Comment:
If async here not needed, we can remove this await i think

Reply:
Ah I should probably just do the `await new Promise(process.nextTick), which allows the async operation to complete. Will update.